components/server/src/orgs/organization-service.ts

```diff
@@ -30,6 +30,37 @@ export class OrganizationService {
         @inject(IAnalyticsWriter) private readonly analytics: IAnalyticsWriter,
     ) {}
 
+    async listOrganizations(
+        userId: string,
+        req: {
+            offset?: number;
+            limit?: number;
+            orderBy?: keyof Organization;
+            orderDir?: "asc" | "desc";
+            searchTerm?: string;
+        },
+    ): Promise<{ total: number; rows: Organization[] }> {
+        const result = await this.teamDB.findTeams(
+            req.offset || 0,
+            req.limit || 50,
+            req.orderBy || "creationTime",
+            req.orderDir === "asc" ? "ASC" : "DESC",
+            req.searchTerm,
+        );
+
+        await Promise.allSettled(
```

Comment:
Why allSettled and not .all? I chose the later, to not cover any errors. :thinking: